jaffle_shop/models/staging/src_seed/_models.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+
+models:
+  - name: stg_customers_pii
```

Comment:
We're missing a sensitive config on this _PII table (how do we know to put this model in consumer_sensitive db?)